examples/3d/motion_blur.rs

```diff
@@ -21,9 +26,11 @@ fn setup_camera(mut commands: Commands) {
         // Motion blur requires the depth and motion vector prepass, which this bundle adds.
         // Configure the amount and quality of motion blur per-camera using this component.
         MotionBlurBundle {
+            #[allow(clippy::needless_update)]
```

Comment:
why is this needed?

Reply:
I'm using `..default()` to fill in the padding value `MotionBlur`, but the padding field doesn't exist in native builds so this raises an error.

I could do the padding manually and/or gate the `..default()` line with `#[cfg(all(feature = "webgl", target_arch = "wasm32"))]` if that's preferable.

Reply:
Ah, got it, makes sense. Maybe using the cfg would make that more explicit yeah and avoid someone potentially removing something that doesn't appear necessary on the surface.

Reply:
> ... gate the `..default()` line with `#[cfg(all(feature = "webgl", target_arch = "wasm32"))]` if that's preferable.

Webgpu and webgl2 aren't technically mutually exclusive now (since the wgpu 0.19 upgrade that was pre-bevy 0.13, having both the webgl and webgpu features enabled forces webgpu). If we don't need the padding on webgpu then the cfg should be `#[cfg(all(feature = "webgl", target_arch = "wasm32", not(feature = "webgpu")))]`